include/oneapi/dpl/pstl/hetero/dpcpp/sycl_defs.h

```diff
@@ -37,36 +37,40 @@
 
 // Combine SYCL runtime library version
 #if defined(__LIBSYCL_MAJOR_VERSION) && defined(__LIBSYCL_MINOR_VERSION) && defined(__LIBSYCL_PATCH_VERSION)
-#    define _ONEDPL_LIBSYCL_VERSION                                                                                    \
+#    define _ONEDPL_DPCPP_LIBSYCL_VERSION                                                                              \
         (__LIBSYCL_MAJOR_VERSION * 10000 + __LIBSYCL_MINOR_VERSION * 100 + __LIBSYCL_PATCH_VERSION)
 #else
-#    define _ONEDPL_LIBSYCL_VERSION 0
+#    define _ONEDPL_DPCPP_LIBSYCL_VERSION 0
 #endif
```

Comment:
> _ONEDPL_LIBSYCL_VERSION was renamed to _ONEDPL_DPCPP_LIBSYCL_VERSION as the definition uses macros specific to DPC++ SYCL implementation.

Does not the same library come with the intel/llvm SYCL compiler as well?
I personally do not consider "libsycl" as a name for any SYCL runtime, but rather the one of DPC++ and intel/llvm.